root/static/scripts/common/entity.js

```diff
@@ -27,7 +27,9 @@ import {
 } from './immutable-entities';
 import linkedEntities from './linkedEntities';
 import MB from './MB';
+import bracketed from './utility/bracketed';
 import clean from './utility/clean';
+import formatLabelCode from '../../../utility/formatLabelCode';
```

Comment:
This should be in a separate import group (does your IDE allow checking for eslint issues? this would have shown as a warning when running eslint on the file)

Reply:
looks like my eslint config was broken, I see it now